internal/node/node_launcher.sh

```diff
@@ -129,8 +129,19 @@ for ARG in "${ALL_ARGS[@]}"; do
   esac
 done
 
+# Note: Bash does not forward any termination signals to any child process when running in docker
+# so need to manually trap and forward the signals
+_term() {
+  kill -TERM "$child" 2>/dev/null
+}
+
+trap _term SIGTERM
+trap _term SIGINT
```

Comment:
Why not do this after bringing up the subprocess? (Or declare the `child` variable first.)